js/wp-seo-admin-306.js

```diff
@@ -81,10 +81,29 @@ jQuery( document ).ready( function() {
 		jQuery( '#' + activeTab + '-tab' ).addClass( 'nav-tab-active' );
 
 		jQuery( '.nav-tab-active' ).click();
+
+		initSelect2();
 	}
 );
 
 /**
+ * Adds select2 for selected fields.
+ */
+function initSelect2() {
+	// Select2 for General settings: your info: company or person. Width is the same as the width for the other fields on this page.
+	jQuery( '#company_or_person' ).select2( { width: '400px' } );
```

Comment:
In my opinion it would be better to put the config in a variable, because it is possible the width of the other fields can be changed and in that case you've to change it on four places.